eclipse-theia/theia
# workspace: update label-provider for multi-root

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/10237

The pull-request updates the `WorkspaceUriLabelProviderContribution` to better support **multi-root** workspaces for `getLongName`. Previously, the label-provider would return the relative path to the workspace root which did not work well for certain cases (displaying an editor's path) and will now display the full path instead.

#### How to test


1. start the application
2. open a workspace - open editors - confirm their paths are relative to the root when hovering over the editor tab labels
3. add a folder to the workspace (creating a multi-root)
4. open editors from both roots
5. confirm the full path is displayed when hovering over the editor tab labels

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
